charts/shoot-core/charts/vpn-shoot/templates/deployment.yaml

```diff
@@ -35,6 +35,19 @@ spec:
       tolerations:
       - key: CriticalAddonsOnly
         operator: Exists
+{{- if .Values.initContainers }}
+      initContainers:
+      {{- range $index, $ctr := .Values.initContainers }}
+      - name: init-{{ $index }}
+        image: {{ $ctr.image }}
+        {{- if $ctr | .command }}
```

Comment:
```suggestion
        {{- if $ctr.command }}
```

Reply:
Oh oops. I originally wrote them in with pipes as placeholders, then changed them back, but forgot one. Thanks @rfranzke !